<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Database_ScreenLayouts</class>
 <widget class="QWidget" name="Database_ScreenLayouts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>892</width>
    <height>539</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>892</width>
    <height>539</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Screen Layouts Tab</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_icons.qrc">
    <normaloff>:/icons/openrpgmaker.png</normaloff>:/icons/openrpgmaker.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">.QFrame
{
	border-width: 1px;
	border-style: solid;
	border-color: palette(window-text);
}

.QFrame&gt;QLabel
{
	border-width: 0px;
	background-color: palette(window-text);
	color: palette(window);
	font-weight: bold;
}

.QFrame&gt;QCheckBox
{
	background-color: palette(window-text);
	border-width: 0px;
	border-style: none;
	color: palette(window);
	spacing: 3px;
	font-weight: bold;
	padding: 0px 0px 0px 3px;
}

.QFrame&gt;QCheckBox::indicator
{
	min-width: 8px;
	max-width: 8px;
	min-height: 8px;
	max-height: 8px;
	width: 8px;
	height: 8px;
	border-width: 2px;
	border-radius: 0px;
	border-style: ridge;
	border-color: palette(mid);
}

.QFrame&gt;QCheckBox::indicator:unchecked
{
	background-color: palette(base);
}

.QFrame&gt;QCheckBox::indicator:checked
{
	background-color: palette(window-text);
}

.QFrame&gt;QCheckBox::indicator:hover
{
	border-color: palette(highlight);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Layouts</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="twLayouts">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wMenuName" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Name:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leMenuName"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>631</width>
       <height>539</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>631</width>
       <height>539</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="ScreenLayouts_BattleLayout" name="wBattleLayout"/>
     <widget class="ScreenLayouts_TitleMenu" name="wTitleMenuLayout"/>
     <widget class="ScreenLayouts_PopupMenu" name="wPopupMenuLayout"/>
     <widget class="ScreenLayouts_MainMenu" name="wMainMenuLayout"/>
     <widget class="QWidget" name="wBlankPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="aNewMainMenu">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New Main Menu</string>
   </property>
   <property name="toolTip">
    <string>New Main Menu</string>
   </property>
  </action>
  <action name="aDeleteMenu">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/editdelete.png</normaloff>:/icons/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Menu</string>
   </property>
   <property name="toolTip">
    <string>Delete Menu</string>
   </property>
  </action>
  <action name="aNewPopupMenu">
   <property name="icon">
    <iconset resource="gui_icons.qrc">
     <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New Popup Menu</string>
   </property>
   <property name="toolTip">
    <string>New Popup Menu</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenLayouts_BattleLayout</class>
   <extends>QWidget</extends>
   <header>screenlayouts_battlelayout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenLayouts_PopupMenu</class>
   <extends>QWidget</extends>
   <header>screenlayouts_popupmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenLayouts_MainMenu</class>
   <extends>QWidget</extends>
   <header>screenlayouts_mainmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenLayouts_TitleMenu</class>
   <extends>QWidget</extends>
   <header>screenlayouts_titlemenu.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
